test/README.md

```diff
@@ -0,0 +1,24 @@
+# Test framework for writing BDD tests using Protractor and Cucumber
+
+The framework Protractor, Cucumber.js and Protractor-Cucumber-Framework which are compatible with each other. 
+
+# Getting started
+
+
+1) Install protractor, cucumber and dependencies
+	
+		npm install
+
+2) Update WebDriver-Manager to get latest binaries
+
+		./node_modules/protractor/bin/webdriver-manager update
+	 
+3) Launch test with gulp
+   	
+		gulp protractor
```

Comment:
I suggest creating a new npm script in `package.json` to call this perhaps `npm run e2e`?